package io.soboro.supreme.storage.db.core.entity.user.exception

class InvalidEmailException : RuntimeException("invalid email has been occurred")
